src/django_components/component_registry.py

```diff
@@ -130,7 +132,7 @@ class InternalRegistrySettings(NamedTuple):
 # We keep track of all registries that exist so that, when users want to
 # dynamically resolve component name to component class, they would be able
 # to search across all registries.
-all_registries: List["ComponentRegistry"] = []
+ALL_REGISTRIES: List[ReferenceType["ComponentRegistry"]] = []
```

Comment:
Also the global list of all created ComponentRegistries is storing now weakrefs instead of strong refenrences.